sdk/deviceupdate/iot-device-update-rest/samples-dev/getDevice.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
@dpokluda  Thank you so much preparing this PR! Could you help run below commands then commit your change? 
```shell
cd ${PROJECT_ROOT}
npx dev-tool samples publish -f
rushx format
```

The `dev-tool samples publish` will generate both JavaScript and TypeScript workable samples from `samples-dev` to `samples` folder. The `rushx format` could format the code to fix build errors.  
We also mentioned these commands in [get started doc](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/RLC-quickstart.md#format-both-the-generated-code-and-manual-code).

Reply:
I did that. Should be fine now.